react/week2/todo-app/src/App.js

```diff
@@ -1,10 +1,9 @@
 import './App.css';
 import getTodos from './getTodos';
-import React from "react";
+import React, { useEffect } from 'react';
 
 const todos = getTodos()
```

Comment:
It's possible to simply "export" the const todos itself without wrapping it in a function like so:

export const todos = []

Reply:
I didnt know that. Thank you 👍